README.md

```diff
@@ -1,11 +1,25 @@
 # Airflow DAGs for dbt
 
-> The code in this repository is meant to accompany [this blog post](https://astronomer.io/blog/airflow-dbt-1) on beginner and advanced implementation concepts at the intersection of dbt and Airflow.
+> The code in this repository is meant to accompany [this blog post](https://astronomer.io/blog/airflow-dbt-1) on
+> beginner and advanced implementation concepts at the intersection of dbt and Airflow.
+
+## To run these DAGs locally:
 
-To run these DAGs locally:
 1. Download the [Astro CLI](https://github.com/astronomer/astro-cli)
 2. Download and run [Docker](https://docs.docker.com/docker-for-mac/install/)
 3. Clone this repository and `cd` into it.
 4. Run `astro dev start` to spin up a local Airflow environment and run the accompanying DAGs on your machine.
+5. **Run the `dbt_basic` DAG first** as this contains a `dbt seed` command that loads sample data into the database. You can then
+run the other two DAGs any time after without having to load the data again.
+
```

Comment:
Hi @spbail, would it be possible to include the `dbt seed` tasks in every DAG and add comments that the tasks aren't necessary to include if other DAGs in the repo will be executed first?  Since we would love for users to be able to run any DAG in the repo out of the box, it would be great to not have internal dependencies.  If there is another way to do this as well feel free to implement.

Reply:
I added the dbt_seed task to the other DAGs too. It's a little redundant in the ELT example with the singer taps, but I think the note in the README and the inline comment are sufficient to explain what this does.